stdlib/2/os/__init__.pyi

```diff
@@ -10,7 +10,7 @@ from mypy_extensions import NoReturn
 error = OSError
 name = ...  # type: str
 
-class _Environ(MutableMapping[str, str]):
+class _Environ(MutableMapping[Union[str, unicode], Union[str, unicode]]):
```

Comment:
This is potentially problematic, since this may break a lot of existing code using `os.environ` that assumes that it only has `str` keys and values (and there are other potential issues as well). The other changes in this PR are more straightforward in the sense that they only make the stubs accept more things. Can you leave this change out from the PR and create a separate issue where we can discuss this?